### PR TITLE
:memo: iTop security policy

### PR DESCRIPTION
**Pull request by piRGoif**
GitHub doc reference : https://help.github.com/en/categories/managing-security-vulnerabilities

Some examples :
https://github.com/electron/electron/blob/master/SECURITY.md
https://github.com/standard/standard/blob/master/SECURITY.md
https://github.com/TryGhost/Ghost/blob/master/SECURITY.md
https://github.com/symfony/symfony/security/policy

We should create security@combodo.com !

This will be reviewed by both R&D and products team.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# 🔒 Reporting vulnerabilities
+
+We take all security bugs seriously. Thank you for improving the security of iTop ! We appreciate your efforts and
```
Comment (Molkobain) [inline SECURITY.md]:
```suggestion
We take all security bugs seriously. Thank you for improving the security of iTop! We appreciate your efforts and
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+You can send us a standard given / then / when, including iTop version, impacts, and maybe installed modules or datas if they are needed to reproduce.
+
+### Dependencies vulnerabilities
+Report security bugs in third-party modules to the person or team maintaining the module, and notify us of the report by sending a mail to security@combodo.com.
+
+
+
+## 📆 Disclosure Policy
+
+Report sent to us will be acknowledge within the week. Then, a developer will be assigned to it and will :
```
Comment (Molkobain) [inline SECURITY.md]:
```suggestion
Report sent to us will be acknowledge within the week. Then, a developer will be assigned to it and will:
```

**Comment by bruno-ds** [comment]
as seen with @Rorcha07 :
 - contact email should be more precise (eg: "itop-security@combodo.com")
 - a (fun) reward may be added, examples:
   - iTop sticker + iTop pencil
   - our eternal gratitude
   - the contributor name in the list of the security contributors
  -  it is maybe not very clear the the points listed in `Disclosure Policy` will be realized by a combodo member, not the discloser

**Comment by piRGoif** [comment]
>     * it is maybe not very clear that the the points listed in `Disclosure Policy` will be realized by a combodo member, not the discloser

I've just pushed a modification.

For the 2 other points, don't forget them and bring them to our attention during the product meeting ;)

**Comment by piRGoif** [comment]
Will discussed further during week 28, stay tuned :)

**Comment by piRGoif** [comment]
Review done, email address created.
Last check on next monday with the teams, and we'll be good to go O:)

**Pull request merged**